scripts/build.js

```diff
@@ -64,7 +65,11 @@ async function run() {
           stdio: 'inherit'
         }
       )
+      console.log()
```

Comment:
Why here a empty `console.log`

Reply:
To have an extra blank line in pnpm run build --withTypes